minor cleanup and improvements

Pokebunny opened the pull request.

sleepywitch committed:
Update .gitignore

basic -gitignore update

sleepywitch committed:
initial development

initial development includes functional command. pretty-fying is needed

sleepywitch committed:
Update classic_teams.py

sleepywitch committed:
Classic Team Prettier

Code Updates to make the classic team commands prettier

sleepywitch committed:
Updating quotes

More quotes and better quotes

sleepywitch committed:
Add DingerCity

Adds 2 of the 4 dinger city classics. The first and second ones. Third is 12 team and fourth is BB. Maybe add a BB classic command sometime

sleepywitch committed:
always test code plz

More adjustments from testing

Pokebunny committed:
minor cleanup and improvements

Pokebunny committed:
minor cleanup and improvements

Pokebunny committed:
minor cleanup and improvements

Pokebunny committed:
minor cleanup and improvements

Pokebunny committed:
minor cleanup and improvements

Pokebunny committed:
Merge remote-tracking branch 'origin/feature/cleanup' into feature/cleanup

Pokebunny committed:
Merge pull request #72 from malonek1/feature/classic-teams

Feature/classic teams

Pokebunny committed:
Merge remote-tracking branch 'origin/feature/cleanup' into feature/cleanup

sleepywitch reviewed (dismissed):
Looks good from the cursory glance I gave it.
im gonna trust you on this one!

MORImementos committed:
Update ladders.py

Update from s5 > s6

MORImementos committed:
Update ladders.py

update new season tags and add prev season constants in case want to add those in

CNace13 committed:
Merge pull request #74 from MORImementos/master

Update tag constants

MORImementos committed:
Update pitching_stat_calcs.py

add pitching stats

MORImementos committed:
Update web_stat_lookup.py

add pitching commands

MORImementos committed:
Update pitching_stat_calcs.py

add sorting filters

MORImementos committed:
Update pitching_stat_calcs.py

Pokebunny committed:
Merge pull request #75 from MORImementos/master

Add pitching stats

Pokebunny committed:
Merge remote-tracking branch 'origin/feature/cleanup' into feature/cleanup

Pokebunny committed:
catchup

Pokebunny committed:
catchup

sleepywitch reviewed (dismissed):
Approved

sleepywitch reviewed (dismissed):
Reapproving!

Pokebunny committed:
Merge remote-tracking branch 'origin/feature/cleanup' into feature/cleanup

sleepywitch commented on main/services/random_functions.py:
```diff
@@ -49,7 +49,7 @@ def rfRandomCharacter():
 # Picks a random stadium
 def rfRandomStadium():
-    stadiums = ["Mario Stadium", "Peach Garden", "Wario Palace", "Yoshi Park", "Gharial Stadium", "Bowser Castle"]
+    stadiums = ["Mario Stadium", "Peach Garden", "Wario Palace", "Yoshi Park", "DK Jungle", "Bowser Castle"]
```
So glad you did this lol

sleepywitch reviewed (dismissed):
Looks good still!

(Anytime you commit or merge or do anything to the code base it's gonna dismiss my review)
